src/Kestrel.Core/CoreStrings.resx

```diff
@@ -500,4 +500,7 @@ For more information on configuring HTTPS see https://go.microsoft.com/fwlink/?l
   <data name="Http2NotSupported" xml:space="preserve">
     <value>HTTP/2 support is experimental, see https://go.microsoft.com/fwlink/?linkid=866785 to enable it.</value>
   </data>
+  <data name="WritingToResponseAfterResponseEnded" xml:space="preserve">
+    <value>Cannot write to the response because the response has ended.</value>
```

Comment:
Wordsmithing time! (I put this text in the bug but I wasn't 100% with it happy yet either ;)). Having `response` in there twice is kinda awkward-sounding.

How about `Cannot write to the response body because the response has completed.` It still has `response` twice, but adding `body` makes it clear that there are two things in play here (the "response" and the "response body")

Reply:
How about "Cannot write to the response body, the response has completed"?